CMakeLists.txt

```
@@ -374,6 +373,7 @@ ExternalProject_Add( chai
                                 -D CMAKE_POSITION_INDEPENDENT_CODE=${CMAKE_POSITION_INDEPENDENT_CODE}
                                 -D ENABLE_RAJA_PLUGIN=ON
                                 -D RAJA_DIR=${RAJA_DIR}/share/raja/cmake
```

```suggestion
                                -D RAJA_DIR=${RAJA_DIR}/lib/cmake/raja
```
The location of the `raja-config.cmake` file has changed